op-guide/location-awareness.md

```diff
@@ -0,0 +1,84 @@
+# 跨机房部署方案
+
+## 概述
+
+PD 能够根据 TiKV 集群的拓扑结构进行调度，使得 TiKV 的容灾能力最大化。
+
+阅读本章前，请先确保阅读 [Binary 部署方案](./binary-deployment.md) 和 [Docker 部署方案](./docker-deployment.md)。
+
+## TiKV 上报拓扑信息
+
+可以通过 TiKV 的启动参数或者配置文件来让 TiKV 上报拓扑信息给 PD。
+
+假设拓扑结构分为三级：zone > rack > host，可以通过 labels 来指定这些信息。
+
+启动参数：
+
+```
+tikv-server --labels zone=<zone>,rack=<rack>,host=<host>
```

Comment:
I think `--label zone=a --label rack=b --label host=c` in command flag and toml labels in config file may be better. 

@andelf Seem that you can support https://github.com/pingcap/tikv/issues/1452, but we should consider compatibility.